scripts/FRI_soundness_list_decoding.sage

```diff
@@ -0,0 +1,286 @@
+from math import log2
+
+
+def commit_soundness_error_list(field_bits, lde_domain_bits, trace_domain_bits, fold_arities, num_polys, m, quotient_degree=0):
+    # Returns the log2 soundness error of the FRI commit phase
+
+    # Parameters:
+    #    field_bits: The number of bits of the field we are drawing the challenges from.
+    #    lde_domain_bits: The log2 of the low-degree extension domain.
+    #    trace_domain_bits: The log2 of the trace domain.
+    #    fold_arities: A list of the log2 of the folding factors used in the FRI protocol.
+    #    num_polys: The degree of the batching plus 1 e.g. for linear batching it is equal to 2.
+    #    m: The proximity paramater to the Johnson bound.
+    #    quotient_degree: The maximal degree of the denominators appearing in the evaluation quotients.
```

Comment:
Why do we set `quotient_degree = 0` as default value?

Reply:
This is because for `quotient_degree = 0` we cover theorem 2 which can be considered as the default case.
`quotient_degree = 2` covers theorem 8.